addons/spectator/ui.hpp

```diff
@@ -18,7 +18,7 @@ class GVAR(display) {
     movingEnable = 0;
     closeOnMissionEnd = 1;
     
-    onLoad = QUOTE(SETUVAR(GVAR(display),_this select 0));
+    onLoad = QUOTE(with uiNameSpace do {GVAR(display) = _this select 0};);
```

Comment:
Shouldn't it rather be `uiNamespace setVariable...`?

Reply:
same effect either way,
I like this method because there are no arrays to deal with